src/misc.jl

```diff
@@ -81,12 +81,107 @@ end
+    column_width = 80
+    taxa = [ node.label for node in POTleaves(t)] #need leaves in Post-Order Traversal
+    max_label_width = maximum([length(taxon) for taxon in taxa])
+    drawing_width = column_width - max_label_width - 1
+    drawing_height = 2 * length(taxa) - 1
 
+    function get_col_positions(t::Tree)
+        depths = [distance_from_root(node) for node in values(t.lnodes)]
+        # If there are no branch lengths, assume unit branch lengths
+        if maximum(depths) == 0
+            depths = [node_depth(node) for node in values(t.lnodes)]
+        end
+        # Potential drawing overflow due to rounding -- 1 char per tree layer
+        fudge_margin = ceil(Int64, log2(length(taxa)))
+        cols_per_branch_unit = (drawing_width - fudge_margin) / maximum(depths)
+        return Dict(zip(keys(t.lnodes), round.(Int64,depths*cols_per_branch_unit .+2.0)))
+    end
 
+    function get_row_positions(t::Tree)
+        positions = Dict{Any, Int64}(zip(taxa, 2 *(1:length(taxa)) ) )
+        function calc_row(clade::TreeNode)
+            for subclade in clade.child
+                if !haskey(positions, subclade.label)
+                    calc_row(subclade)
+                end
+            end
+            positions[clade.label] = floor(Int64, (positions[clade.child[1].label] + positions[clade.child[end].label])/2)
+        end
+
+        calc_row(t.root)
+        return positions
+    end
+
+    col_positions = get_col_positions(t)
+    row_positions = get_row_positions(t)
+    char_matrix = [[" " for x in 1:(drawing_width+1)] for y in 1:(drawing_height+1)]
+
+    function draw_clade(clade::TreeNode, startcol::Int64)
+        thiscol = col_positions[clade.label]
+        thisrow = row_positions[clade.label]
+        # Draw a horizontal line
+        for col in startcol:thiscol
+            char_matrix[thisrow][col] = "_"
+        end
+        if !isempty(clade.child)
+            # Draw a vertical line
+            toprow = row_positions[clade.child[1].label]
+            botrow = row_positions[clade.child[end].label]
+            for row in (toprow+1):botrow
+                char_matrix[row][thiscol] = "|"
+            end
+            # NB: Short terminal branches need something to stop rstrip()
+            if (col_positions[clade.child[1].label] - thiscol) < 2
+                char_matrix[toprow][thiscol] = ","
+            end
+            # Draw descendents
+            for child in clade.child
+                draw_clade(child, thiscol + 1)
+            end
+        end
+    end
+    draw_clade(t.root, 1)
+    # Print the complete drawing
+    for i in 1:length(char_matrix)
+        line = join(char_matrix[i])
+        # Add labels for terminal taxa in the right margin
+        if i % 2 == 0
+            line = line * " " * taxa[round(Int64, i/2)]
+        end
+        println(line)
+    end
+end
+
+"""
+    distance_from_root(node::TreeNode)
+
+    in added branch lengths
+"""
+function distance_from_root(node::TreeNode)
```

Comment:
There is already a `divtime` function that computes the branch length distance between two nodes (now that I think about it, it should be renamed to `distance`). One would call it like `divtime(node, root)`. 
If you think it's practical to have a separate `distance_to_root` function, you could move it to `methods.jl`, along with other "distance" functions.